src/components/organisms/EntityList/EntityListFilters/LabelsFilter/LabelsFilter.tsx

```diff
@@ -161,6 +163,7 @@ const LabelsFilter: React.FC<FilterProps> = props => {
 
   const menu = (
     <StyledFilterMenu
+      onClick={onMenuClick}
```

Comment:
I remember this prop was removed recently, is it appears to be needed?

Reply:
Good point, I will adjust it then on some other PR in Cloud